docs/guide/testing.md

```diff
@@ -233,3 +233,47 @@ To use component previews:
 # config/application.rb
 config.view_component.preview_paths << "#{Rails.root}/spec/components/previews"
 ```
+
+## Testing Interactive Components
+
+Use the `with_rendered_component_path` helper method in your system tests to test interactivity of components.
+
+```rb
+require "test_helper"
+
+class ViewComponentSystemTest < ViewComponent::SystemTestCase
+  driven_by :cuprite
+
+  def test_simple_js_interaction_in_browser_without_layout
+    with_rendered_component_path(SimpleJavascriptInteractionWithJsIncludedComponent.new) do |page|
+      visit page
```

Comment:
`page` is a reserved word in most system tests:

```suggestion
    with_rendered_component_path(SimpleJavascriptInteractionWithJsIncludedComponent.new) do |path|
      visit(path)
```

Reply:
Good point!